compiler/luci-interpreter/src/kernels/Sub.cpp

```diff
@@ -82,6 +89,44 @@ void Sub::evalFloat() const
   }
 }
 
+template <typename T> void Sub::evalInteger() const
+{
+  tflite::ArithmeticParams params{};
+  if (std::is_same<T, int32_t>::value)
+  {
+    int32_t activation_min{};
+    int32_t activation_max{};
+    calculateActivationRange(_params.activation, &activation_min, &activation_max);
+    params.quantized_activation_min = activation_min;
+    params.quantized_activation_max = activation_max;
+  }
+  else
+  {
+    assert((std::is_same<T, int64_t>::value));
+    int64_t activation_min{};
+    int64_t activation_max{};
+    calculateActivationRange(_params.activation, &activation_min, &activation_max);
+    params.int64_activation_min = activation_min;
+    params.int64_activation_max = activation_max;
+  }
```

Comment:
(optional) May be simplify a bit?

```suggestion
calculateActivationRange(_params.activation, &params.quantized_activation_min, &params.quantized_activation_max);
calculateActivationRange(_params.activation, &params.int64_activation_min, &params.int64_activation_max);
```

Reply:
let's refactor this in separate PR, I think we can find same code in other operations